src/operator/nn/mkldnn/mkldnn_convolution.cc

```diff
@@ -88,26 +91,23 @@ static mkldnn::convolution_backward_data::primitive_desc GetConvBwdData(
   auto weight_md = GetWeightDesc(weights, param.num_group);
   auto out_md = GetMemDesc(output);
   auto engine = CpuEngine::Get()->get_engine();
+  CHECK_GE(param.stride.ndim(), 2U);
```

Comment:
should this be CHECK_EQ ?

Reply:
after ONNX is fixed, this should be CHECK_EQ. I didn't know if ONNIX would be fixed when I submitted the PR.

Reply:
Please disregard my comment. I think this change shouldnt depend on whether onnx is fixed or not. `CHECK_GE` looks good to make it consistent with existing behavior.